src/coreclr/jit/target.h

```diff
@@ -305,56 +364,69 @@ struct regMaskTP
 
     SingleTypeRegSet GetPredicateRegSet() const
     {
+#ifdef HAS_MORE_THAN_64_REGISTERS
+        return getHigh();
+#else
         return getLow();
+#endif
     }
 
-    void RemoveRegNumFromMask(regNumber reg);
+    void operator|=(const regMaskTP& second)
+    {
+        low |= second.getLow();
+#ifdef HAS_MORE_THAN_64_REGISTERS
+        high |= second.getHigh();
+#endif
+    }
 
-    bool IsRegNumInMask(regNumber reg);
+    void operator^=(const regMaskTP& second)
+    {
+        low ^= second.getLow();
+#ifdef HAS_MORE_THAN_64_REGISTERS
+        high ^= second.getHigh();
+#endif
+    }
+
+    void operator&=(const regMaskTP& second)
+    {
+        low &= second.getLow();
+#ifdef HAS_MORE_THAN_64_REGISTERS
+        high &= second.getHigh();
+#endif
+    }
+
+    void operator|=(const regNumber reg);
+    void operator^=(const regNumber reg);
 };
 
-static regMaskTP operator^(regMaskTP first, regMaskTP second)
+static regMaskTP operator^(const regMaskTP& first, const regMaskTP& second)
```

Comment:
Does this make a difference?

Reply:
don't think so. I just wanted to make it consistent like other methods.